studio/src/components/analytics/trace-details.tsx

```diff
@@ -81,10 +84,17 @@ export const TraceDetails = ({ ast }: { ast: GraphQLSchema | null }) => {
     );
 
     if (routerSpan) {
-      set(
-          routerSpan.attributes?.operationContent || "",
-          routerSpan.attributes?.operationVariables || "",
-      ).catch((e) => console.error("Error during formatting", e));
+      const content = routerSpan.attributes?.operationContent;
+        const variables = routerSpan.attributes?.operationVariables;
+
+      setContent(content);
```

Comment:
Please set the content once, this in the above function catch block, so there is no rerender.

Reply:
Done